cloud/parameterstore/ssm_client.go

```diff
@@ -114,19 +113,59 @@ func (c *ssmClientImpl) GetParametersSimple(ctx context.Context, input *ssm.GetP
 }
 
 func (c *ssmClientImpl) GetParameters(ctx context.Context, input *ssm.GetParametersInput) ([]*ssm.GetParametersOutput, error) {
-	// TODO (DEVPROD-9391): follow-up PR should handle batching requests due
-	// to limit of 10 parameters per API call.
-	output, err := retrySSMClientOp(ctx, "GetParameters", c.client, input, func(ctx context.Context, client *ssm.Client, input *ssm.GetParametersInput) (*ssm.GetParametersOutput, error) {
+	const maxParamsPerRequest = 10
```

Comment:
should this just be a const at the top of the file or do some operations use different max batch sizes? Refactoring this out could be helpful since the `retryBatchSSMClientOp` signature is long.

Reply:
I could only find 2 operations that mention a batch size, but I guess it's fine to move into a constant for now.

And yeah, unfortunately generics that use generic types can end up with long method signatures, there's not really a way to shorten it since it's doing a lot :\